main.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:400,900');
+@import url('https://fonts.googleapis.com/css?family=Raleway:400,600');
+
+html {
+  font-size: 16px;
+}
+body {
+  font-size: 1rem;
+  font-family: 'Roboto', sans-serif;
+}
+
+
+
+/* Programme Builder CELLS */
+
+.prog-cell {
+  border: 1px solid #979797;
```

Comment:
also needs 
`padding: 0;` because it's given a little padding by the nature of being a `<td>`